testing/src/main/java/com/linecorp/decaton/testing/TestUtils.java

```diff
@@ -62,6 +72,7 @@ private static Properties defaultProducerProps(String bootstrapServers) {
     }
 
     public static final String DEFAULT_GROUP_ID = "test-group";
+    public static final Duration DEFINITELY_TOO_SLOW = Duration.ofSeconds(20);
```

Comment:
> add timeouts of 20s in some places of the tests that were blocking forever, for easier debugging

curious why backtrace provided when test itself times out (`@Test(timeout = xxx)`) isn't sufficient to know at where it stuck?

Reply:
because it was not exactly stuck. Producer and Consumer were running and doing nothing.